widgetssdk/src/main/res/values/new_strings.xml

```diff
@@ -146,6 +146,8 @@
     <string name="call_buttons_chat_badge_value_multiple_items_accessibility_label">@string/glia_call_chat_other_content_description</string>
     <string name="call_buttons_chat_badge_value_single_item_accessibility_label">@string/glia_call_chat_one_content_description</string>
     <string name="general_you">@string/glia_call_visitor_video_on_hold_text</string>
+    <string name="call_visitor_video_flip_to_back_camera_button_accessibility_label">Switch to back camera</string>
```

Comment:
This is outdated, right?

Reply:
Right. For some reason, I couldn't push fixed yesterday. But now it is fixed.